### PR TITLE
updated Swagger to 5.6.3 version

### LINKED ISSUE #115: Set up a swagger
Set up a swagger

https://docs.microsoft.com/ru-ru/aspnet/core/tutorials/web-api-help-pages-using-swagger?view=aspnetcore-3.1

https://aspnetcore.readthedocs.io/en/stable/tutorials/web-api-help-pages-using-swagger.html

### PR DESCRIPTION
**Pull request by ArdeaSt**
Close #115 
Updated Swagger to 5.6.3
Add AuthJwt for Swagger
Add FluentValidation for models in Swagger

### CODE REVIEW

File: EventsExpress/Startup.cs [hunk 1]
```diff
@@ -159,13 +160,42 @@ public void ConfigureServices(IServiceCollection services)
...
+            {
+                Type = ReferenceType.SecurityScheme,
+                Id = "Bearer",
+            },
+            Scheme = "oauth2",
+            Name = "Bearer",
+            In = ParameterLocation.Header,
+        }, new List<string>()
+    },
+});
```
Comment (sand0) [inline EventsExpress/Startup.cs]: fix formatting, pls

File: EventsExpress/Startup.cs [hunk 2]
```diff
@@ -196,18 +226,22 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
...
             {
                 routes.MapRoute(
                     name: "default",
                     template: "{controller}/{action=Index}/{id?}");
             });
+
+            app.UseSwagger(c =>
+            {
+                c.SerializeAsV2 = true;
+            });
```
Comment (sand0) [inline EventsExpress/Startup.cs]: Not sure. Are we really need to use v2?

**Review by sand0 — CHANGES_REQUESTED** [review]

File: EventsExpress/Startup.cs [hunk 3]
```diff
@@ -196,18 +226,22 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
...
+            app.UseSwaggerUI(c =>
+            {
+                c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
```
Comment (iuriikhrystiuk) [inline EventsExpress/Startup.cs]: Name it again as `EventsExpress API` as you did above.

Comment (ArdeaSt) [reply EventsExpress/Startup.cs]: changed

File: EventsExpress/Startup.cs (hunk 2 above)
Comment (ArdeaSt) [reply EventsExpress/Startup.cs]: deleted

File: EventsExpress/Startup.cs (hunk 1 above)
Comment (ArdeaSt) [reply EventsExpress/Startup.cs]: fixed code

**Review by sand0 — APPROVED** [review]

**Pull request merged**